presentation/src/main/java/com/mashup/alcoholfree/presentation/ui/component/SulSulButton.kt

```diff
@@ -0,0 +1,187 @@
+package com.mashup.alcoholfree.presentation.ui.component
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.mashup.alcoholfree.presentation.R
+import com.mashup.alcoholfree.presentation.ui.component.model.SulSulButtonColor
+import com.mashup.alcoholfree.presentation.ui.component.model.SulSulButtonSize
+import com.mashup.alcoholfree.presentation.ui.theme.White
+
+@Composable
+fun SulSulDefaultButton(
+    modifier: Modifier = Modifier,
+    content: String,
+    buttonColor: SulSulButtonColor,
+    buttonSize: SulSulButtonSize,
+) {
+    Box(
+        modifier = modifier
+            .clip(shape = RoundedCornerShape(buttonSize.radius.dp))
+            .background(color = buttonColor.color),
+    ) {
+        Row(
+            horizontalArrangement = Arrangement.Center,
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = modifier.padding(
+                horizontal = buttonSize.horizontalPadding.dp,
+                vertical = buttonSize.verticalPadding.dp,
+            ),
+        ) {
```

Comment:
```suggestion
        Row(
            modifier = modifier.padding(
                horizontal = buttonSize.horizontalPadding.dp,
                vertical = buttonSize.verticalPadding.dp,
            ),
            horizontalArrangement = Arrangement.Center,
            verticalAlignment = Alignment.CenterVertically,
        ) {
```

Nit: modifier가 맨 위에있는게 보기 편한데 왜인지 모르겠어. 관성적으로 맨 위에 두고했었는데.. 기본 설정같은 느낌이라서 일까.. 컨벤션 같은게 있는거면 수정해도 좋을거같고 없으면 놔둬도 될듯!?

Reply:
넵 이건 컨벤션 한번 찾아보겠습니다! 없으면 그래도 바꿀게여. 저는 뭐 다 괜찮습니당